spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -156,7 +156,7 @@ public TaskExecutionController(TaskExplorer explorer, TaskExecutionService taskE
 	public long launch(@RequestParam("name") String taskName,
 			@RequestParam(required = false) String properties,
 			@RequestParam(required = false) String arguments,
-			@RequestParam(required = false, defaultValue = "default") String platformName) {
+			@RequestParam(required = false, defaultValue = "default", name="platform-name") String platformName) {
```

Comment:
how do we know this works?

Reply:
This is our property that we set from the client side as in [here](https://github.com/spring-cloud/spring-cloud-dataflow/pull/2771/files#diff-67f6359ec02f9f7a13fb5c84968a3596R156)